### DISCUSSION TITLE
Does any NPU that supports TensorFlowLite work?

### DISCUSSION THREAD

**Discussion by NoelGisler** [blakeblackshear/frigate#12535] [General]
I found the Rockpi N10 whose NPU supports Tensorflow lite.

Can I just plug it in and it will work? Or is there more to it?

**Comment by NickM-27** [comment]
It has to be directly supported, I believe only rockchip small board computers with built in NPUs are supported

**Reply by NoelGisler** [reply]
It has to be actively supported by Frigate? That makes it a bit harder. Thanks for your help.

I couldn't find a List with NPUs supported by Frigate, do you know how I find out which NPUs are supported?

**Reply by NickM-27** [reply]
Currently NPUs are only supported in 0.14
